platform-coop-toolkit/pcc
# feat: adding community voices stories and listings

## Description

Adds stories to the Community Voices section of the website. Depends on https://github.com/platform-coop-toolkit/pcc-framework/pull/164.

## Steps to test

*From the Admin Interface*
1. Disable any existing redirects for `/stories/` (ex. disable `/stories/ -> /voices/blog/`, `^/stories/*/ > /blog/$1`.
2. Refresh the permalinks: Settings > Permalinks > Save
3. Activate the `pcc` theme with the changes from this pull request if you haven't already.
4. Verify that you have some Stories already by checking `Stories > All Stories`. To add some Stories: follow the instructions detailed [in this Pull Request](https://github.com/platform-coop-toolkit/pcc-framework/pull/164).

*From the Front End*
1. Go to `Community Voices > Community Stories`.
2. Verify that filtering stories by Sectors or Region works as expected.
3. Verify that "Show only stories by:" works as expected.
4. Select a Story. Verify that the video is embedded in the content, and that the content appears as expected.

## Additional information



## Related issues

Depends on https://github.com/platform-coop-toolkit/pcc-framework/pull/164.
Fixes #85
